website/blog/2023-04-01-databend-weekly-87.md

```diff
@@ -0,0 +1,181 @@
+---
+title: "This Week in Databend #87"
+date: 2023-04-01
+slug: 2023-04-01-databend-weekly
+tags: [databend, weekly]
+description: "Stay up to date with the latest weekly developments on Databend!"
+contributors:
+  - name: andylokandy
+  - name: ariesdevil
+  - name: b41sh
+  - name: BohuTANG
+  - name: dantengsky
+  - name: Dousir9
+  - name: drmingdrmer
+  - name: everpcpc
+  - name: jun0315
+  - name: leiysky
+  - name: lichuang
+  - name: PsiACE
+  - name: RinChanNOWWW
+  - name: rkmdCodes
+  - name: SkyFan2002
+  - name: soyeric128
+  - name: sundy-li
+  - name: TCeason
+  - name: Xuanwo
+  - name: xudong963
+  - name: youngsofun
+  - name: zhang2014
+  - name: zhyass
+authors:
+  - name: PsiACE
+    url: https://github.com/psiace
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### Window Function
+
+Aggregate window functions can operate on a group of rows and return a single value for each row in the underlying query. The `OVER` clause specifies how to partition the rows in the result set. When used with `GROUP BY`, aggregate window functions do not collapse rows but instead return all rows in the result set.
+
+```sql
+-- use aggrerate window function
+SELECT date, AVG(amount) over (partition by date)
+FROM BookSold
+
+June 21|544.0
+June 21|544.0
+June 22|454.5
+June 22|454.5
+June 23|643.0
+June 23|643.0
+```
+
+Databend supports all aggregate functions as aggregate window functions.
+
+- [Doc | (draft) Aggregate Window Functions](https://databend.rs/doc/sql-functions/window-functions/aggregate-window-functions)
+- [PR #10700 | feat(window): initial impl window function](https://github.com/datafuselabs/databend/pull/10700).
+
+### Suggest Function Name on Typos
+
+Databend has added an intelligent feature that automatically provide the closest matching item when you enter an incorrect function name.
+
+```sql
+#> select base64(1);
+ERROR 1105 (HY000) at line 1: Code: 1008, displayText = error:
+  --> SQL:1:8
+  |
+1 | select base64(1)
+  |        ^^^^^^^^^ no function matches the given name: 'base64', do you mean 'to_base64'?
```

Comment:
-> No function matches the given name: 'base64'. Did you mean 'to_base64'?

Reply:
This is an error report in a test, I just copied it.